Provis.Core/Services/WorkspaceService.cs

```diff
@@ -237,6 +242,17 @@ public async Task AcceptInviteAsync(int inviteId, string userid)
 
             inviteUserRec.IsConfirm = true;
 
+            var userTaskSpecification = new UserTasks.UserTaskList(userid, inviteUserRec.WorkspaceId);
+            var userTasks = await _userTaskRepository.GetListBySpecAsync(userTaskSpecification);
+
+            if (userTasks != null)
+            {
+                foreach (var userTask in userTasks)
+                {
+                    userTask.Item2.IsUserDeleted = false;
+                }
+            }
```

Comment:
bool already has a false value, should we set it again ?

Reply:
If we leaved from workspace, all workspace task is marked like "is deleted = true". And than if we want to return, we need to return the "is deleted = false" in all workspace tasks